index.html

```diff
@@ -287,13 +287,18 @@ <h3>
  -> https://w3c-ccg.github.io/user/alice/did.json
           </pre>
 
+          <pre class="example nohighlight" title="Creating the DID with a path containing a colon (not recommended but allowed by URI rules)">
+did:web:example.com/example%3Asubdirectory
```

Comment:
```suggestion
did:web:example.com:example%3Asubdirectory
```

Reply:
Ohh I missed this. Definitely agree with @clehner suggestion here.